src/cards/stats-card.js

```diff
@@ -394,7 +394,7 @@ const renderStatsCard = (stats, options = {}) => {
         <circle class="rank-circle-rim" cx="-10" cy="8" r="40" />
         <circle class="rank-circle" cx="-10" cy="8" r="40" />
         <g class="rank-text">
-          ${rankIcon(rank_icon, rank?.level, rank?.percentile)}
+          ${rankIcon(name, rank_icon, rank?.level, rank?.percentile)}
```

Comment:
The githubusercontent requires the username, eg "anuraghazra" it cannot work with the name eg "Anurag Hazra", so you need to pass down the username.